region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -105,18 +105,37 @@
       "name": "Gain Flash Suit (Spikesuit)",
       "requires": [
         {"obstaclesCleared": ["A"]},
-        {"obstaclesNotCleared": ["B"]},
-        {"canShineCharge": {"usedTiles": 29, "openEnd": 0}},
-        {"thornHits": 4},
         "h_spikeSuitThornHitLeniency",
-        "canSpikeSuit",
-        {"shinespark": {"frames": 1, "excessFrames": 1}}
+        {"or": [
+          {"and": [
+            {"obstaclesNotCleared": ["B"]},
+            {"canShineCharge": {"usedTiles": 29, "openEnd": 0}},
+            {"thornHits": 4},
+            "canSpikeSuit",
+            {"shinespark": {"frames": 1, "excessFrames": 1}}
+          ]},
+          {"and": [
+            {"obstaclesNotCleared": ["B"]},
+            {"ammo": {"type": "PowerBomb", "count": 1}},
```

Comment:
The scaling is really bad on this branch.  I read it as a room reset per attempt, and another PB and clearing the hoppers each time.  Probably have to cut it.

Reply:
And it would have to clear B.  So would be a separate strat anyway.